Remove argparse, it's in python-2.7 and robotpkg supports only python >= 2.7
Furthermore, it creates issues in Fedora (pkg_resources.DistributionNotFound)
Last, it's only for pip which we really don't care about here.

--- setup.py~	2018-08-08 22:39:35.000000000 +0200
+++ setup.py	2018-09-14 17:11:50.773394417 +0200
@@ -34,12 +34,6 @@
     'description': 'catkin package library',
     'long_description': 'Library for retrieving information about catkin packages.',
     'license': 'BSD',
-    'install_requires': [
-        'argparse',
-        'docutils',
-        'python-dateutil',
-        'pyparsing',
-    ],
 }
 if 'SKIP_PYTHON_MODULES' in os.environ:
     kwargs['packages'] = []
